.gitignore

```diff
@@ -7,6 +7,7 @@ dist
 .next
 .eslintcache
 .docz
+.vscode/
```

Comment:
let's not add stuff unrelated to this PR scope.

Reply:
no problem